src/docs/styles/custom.css

```diff
@@ -68,3 +68,20 @@ body,
 #app table td {
     text-align: center;
 }
+
+/** Loading container */
+.loading-container {
+    position: relative;
+}
+
+.loading-container .load-state {
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    position: absolute;
+}
+
+.loading-container--loading .britechart {
```

Comment:
Instead of using a variant with --, I would use a SMACCS state, something like .is-loading